### PR TITLE
Fix a failing android test and cleanup

### PR DESCRIPTION
**Pull request by yschimke**
```
okhttp.android.test.OkHttpTest > testRequest[test(AVD) - 10] FAILED 
	org.opentest4j.AssertionFailedError: expected: <200> but was: <404>
	at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
```

### CODE REVIEW

File: android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt [hunk 1]
```diff
@@ -127,26 +127,26 @@ class OkHttpTest(val server: MockWebServer) {
...
-        .sslSocketFactory(clientCertificates.sslSocketFactory(), clientCertificates.trustManager)
-        .build()
+      .sslSocketFactory(clientCertificates.sslSocketFactory(), clientCertificates.trustManager)
+      .build()
 
     val response = client.newCall(request).execute()
 
     response.use {
-      assertEquals(200, response.code)
+      assertEquals(404, response.code)
```
Comment (yschimke) [inline android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt:145]: This is the fix

File: build.gradle.kts [hunk 2]
```diff
@@ -65,8 +65,25 @@ allprojects {
...
+          "indent_size" to "2",
+          "disabled_rules" to "filename"
+        )
+      )
+      trimTrailingWhitespace()
+      endWithNewline()
+    }
+  }
+
+  if (project.name == "android-test") return@subprojects
```
Comment (yschimke) [inline build.gradle.kts:86]: Moved below so at least this check runs on android projects.

File: android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt [hunk 3]
```diff
@@ -550,18 +572,19 @@ class OkHttpTest(val server: MockWebServer) {
...
 
     client.get("https://www.facebook.com/robots.txt")
   }
 
-
   @Test
   fun testCustomSSLSocketFactoryWithoutALPN() {
     enableTls()
 
+    server.enqueue(MockResponse().setBody("abc"))
```
Comment (yschimke) [inline android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt:586]: And this.

**Comment by yschimke** [comment]
Landing - to see a green build on master. cc @swankjesse

File: android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt (hunk 1 above)
Comment (swankjesse) [reply android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt:145]: Alternate fix is changing the URL?

Comment (swankjesse) [reply android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt:145]: (this is fine.)

**Review by swankjesse — COMMENTED** [review]
LGTM!

**Pull request merged**